index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Home
+nav_order: 1
+---
+
+# Overview
+
+The role of the Technical Advisory Committee (TAC) of the Academy Software Foundation (ASWF) is to facilitate communication and collaboration among the Technical Projects. The TAC will be responsible for:
+
+*  Coordinating collaboration among Technical Projects, including development of an overall technical vision for the community;
+*  Making recommendations to the Budget Committee of resource priorities for Technical Projects;
+*  Electing annually a chairperson to preside over meetings, set the agenda for meetings, ensure meeting minutes are taken and who will also serve on the Governing Board as the TAC’s representative (the “TAC Representative”); and
+*  Such other matters related to the technical role of the TAC as may be communicated to the TAC by the Governing Board.
+
+# TAC Members
+
+All members of the broad ASWF community are welcome to participate in TAC activities, such as meetings or online discussion. For voting matters such as project inclusion or graduation, TAC members are composed of:
+
+* One representative appointed by each Premier Member;
+* One representative appointed by the technical oversight body (e.g., a technical steering committee) of each TAC Project; and
+* Up to three experts from the broader motion picture industry (who serve for one-year terms)
+
+Current TAC Members are:
+
+<table>
+    <tr>
+        <th>Representative</th>
+        <th>Appointed By</th>
+        <th>Role</th>
+        <th>Special Role</th>
+        <th>Organization</th>
+    </tr>
+{%- for member in site.data.tacmembers -%}
+    <tr>
+        <td>{{ member["Full Name"] }}</td>
+        <td>{{ member["Appointed By"] }}</td>
+        <td>{{ member["Voting Status"] }}</td>
+        <td>{{ member["Special Role"] }}</td>
+        <td>{{ member["Account Name: Account Name"] }}</td>
+    </tr>
+{%- endfor -%}
+</table>
+
+# Technical Projects
+
+Technical Projects are approved by TAC Members per the [ASWF Project Lifecycle guidelines](https://github.com/AcademySoftwareFoundation/tac/blob/master/process/lifecycle.md). You can submit your project for inclusion as an ASWF Technical Project by completing the [Project Contribution Proposal Template](https://github.com/AcademySoftwareFoundation/tac/blob/master/process/proposal_template.md) and emailing it to tac@lists.awsf.io.
+
+Each of the ASWF projects are open to participation by anyone subject to the governance each project has adopted. If you are looking for a way to contribute to a project, many ASWF projects maintain a list of "good first issues" ( linked to below ).
+
+Current Technical Projects are listed below.
+
+<!-- Embed list of all Open Mainframe Project members -->  
```

Comment:
This comment looks like cut-n-paste from Open Mainframe Project

Reply:
Fixed!